### PR TITLE
Allow specification of the instance's root disk size

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull requests adds an optional input variable specifying the size of the FreeIPA instance's root disk in GB.  The new variable is optional, so it will not break existing code.

See also cisagov/cool-sharedservices-freeipa#42.

## 💭 Motivation and context ##

In some cases it makes sense to have more space than the 8GB size of the AMI.

## 🧪 Testing ##

I have successfully applied these changes to our production COOL environment.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Handy improvement! ⛱️

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ module "ipa1" {
 | nessus\_key\_key | The SSM Parameter Store key whose corresponding value contains the secret key that the Nessus Agent should use when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/key). | `string` | n/a | yes |
 | nessus\_port\_key | The SSM Parameter Store key whose corresponding value contains the port to which the Nessus Agent should connect when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/port). | `string` | n/a | yes |
 | realm | The realm for the IPA server (e.g. EXAMPLE.COM).  Only used if this IPA server IS NOT intended to be a replica. | `string` | `"EXAMPLE.COM"` | no |
+| root\_disk\_size | The size of the IPA instance's root disk in GB. | `number` | `8` | no |
```
Comment (mcdonnnj) [inline README.md]: This is getting into really pedantic turf, but it's supported [by the documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#volume_size).

```suggestion
| root\_disk\_size | The size of the IPA instance's root disk in GiB. | `number` | `8` | no |
```

File: variables.tf [hunk 2]
```diff
@@ -76,6 +76,12 @@ variable "realm" {
   default     = "EXAMPLE.COM"
 }
 
+variable "root_disk_size" {
+  type        = number
+  description = "The size of the IPA instance's root disk in GB."
```
Comment (mcdonnnj) [inline variables.tf]: Just mirroring the previous suggestion.

```suggestion
  description = "The size of the IPA instance's root disk in GiB."
```

**Review by mcdonnnj — APPROVED** [review]
Thanks for getting this together! I had one suggestion around units of measure for your consideration.

**Comment by jsf9k** [comment]
The checks are currently failing due to the Terraform version upgrade.  I will re-run them when the dust settles and they should pass.

**Pull request merged**